CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Make `B2Api.get_bucket_by_id` return populated bucket objects in v2
 * Add proper support of `recommended_part_size` and `absolute_minimum_part_size` in `AccountInfo`
 * Refactored `minimum_part_size` to `recommended_part_size` (tha value used stays the same)
+* Refactored `sync.file.*File` and `sync.file.*FileVersion` to `sync.path.*SyncPath`
```

Comment:
Why added between? The changes should be ordered in the changelog in the same way they were introduced.